source/components/cardContainer/card/headerColumn/sizeForBreakpoints.tests.ts

```diff
@@ -0,0 +1,74 @@
+import { SizeForBreakpoints } from './sizeForBreakpoints';
+
+import { services } from 'typescript-angular-utilities';
+import __string = services.string;
+import __object = services.object;
+
+describe('SizeForBreakpoints', () => {
+	let sizeForBreakpoints: SizeForBreakpoints;
+
+	beforeEach(() => {
+		sizeForBreakpoints = new SizeForBreakpoints(__string.stringUtility, __object.objectUtility);
+	});
+
+	it('should convert an object of breakpoint sizes into a class string', () => {
+		const sizes = {
+			lg: 4,
+			md: 3,
+			sm: 2,
+			xs: 1,
+		};
+		expect(sizeForBreakpoints.getClass(sizes, '')).to.equal('col-xs-1 col-sm-2 col-md-3 col-lg-4 ');
```

Comment:
Ideally I think the view would be the only piece that has any notion of column sizes, and not the code but I know the purpose of the card container is to make that whole process easier so I'm kinda nitpicking and this point. It really doesn't matter to me much; more hypothetical than anything :smiley:. There's also probably an abstraction we could use to decouple the two, but I'm sure it's not worth the overhead. I'm good with it :+1: